src/construction.cpp

```diff
@@ -947,6 +950,37 @@ void construct::done_trunk_plank( const tripoint &p )
     }
 }
 
+void construct::done_grave( const tripoint &p )
+{
+    map_stack its = g->m.i_at( p );
+    for( item it : its ) {
+        if( it.is_corpse() ) {
+            if( it.get_corpse_name().empty() ) {
+                if( it.get_mtype()->has_flag( MF_HUMAN ) ) {
```

Comment:
It seems that if it's not human corpse, then do nothing? We were supposed to bury pets too, but in this case there's no morale changes nor sad words.

Reply:
Actually if your pet animal was named, it falls under further check, that does not check if corpse was a human. I double checked if pet names carry on to their corpses and if it is recognized by the bury code. It is. Will fix the descriptions of morale gain to more general though.

![image](https://user-images.githubusercontent.com/39925111/57395341-2df5eb80-71c8-11e9-95bd-2da29773fe54.png)

So mourn all you want, but name your pet first.
I wish I could say that no in-game dog was harmed in making this feature but... c'est la vie.